pyproject.toml

```diff
@@ -21,9 +21,10 @@ packages = [
 
 
 [tool.poetry.dependencies]
-python = ">=3.8,<3.12"
-dcicutils = "^8.0.0"
-tibanna-ff = ">=3.2.0"
+python = ">=3.8.1,<3.13"
```

Comment:
I would not exclude 3.8.0 here if there is no good reason. Technically that would be a breaking change.

Reply:
Okay, will have to make dcicutils be >=3.8 too which requires downgrading flake8, etc; I can see how that goes.